feature/org.wso2.carbon.identity.common.feature/resources/identity-event.properties

```diff
@@ -27,4 +27,6 @@ account.lock.handler.subscription.5=POST_UPDATE_USER_CLAIMS_PUT
 account.lock.handler.subscription.6=POST_UPDATE_USER_CLAIMS_PATCH
 module.name.2=emailSend
 emailSend.subscription.1=TRIGGER_NOTIFICATION
-
+module.name.3=ask.password.using.email
+ask.password.using.email.subscription.1=POST_ADD_USER
+ask.password.using.email.subscription.2=POST_ADD_USER_BY_DOMAIN
```

Comment:
@sagara-gunathunga To discuss: this property naming specification seems obscure, we need to come up with some good convention.